content/en/docs/contribute/new-content/blogs-case-studies.md

```diff
@@ -12,35 +12,73 @@ weight: 30
 Anyone can write a blog post and submit it for review.
 Case studies require extensive review before they're approved.
 
-
-
 <!-- body -->
 
-## Write a blog post
-
-Blog posts should not be
-vendor pitches. They must contain content that applies broadly to
-the Kubernetes community. The SIG Docs [blog subproject](https://github.com/kubernetes/community/tree/master/sig-docs/blog-subproject) manages the review process for blog posts. For more information, see [Submit a post](https://github.com/kubernetes/community/tree/master/sig-docs/blog-subproject#submit-a-post).
+## The Kubernetes Blog
 
-To submit a blog post, you can either:
+The Kubernetes blog is used by the project to communicate new features, community reports, and any news that might be relevant to the Kubernetes community. 
+This includes end users AND developers. 
+Generally speaking the blog tends to skew towards things that happen in the core project, however the community is always interested in things that are happening in the Kubernetes ecosystem, so we encourage you to consider submitting content to the blog. 
 
-- Use the
-[Kubernetes blog submission form](https://docs.google.com/forms/d/e/1FAIpQLSdMpMoSIrhte5omZbTE7nB84qcGBy8XnnXhDFoW0h7p2zwXrw/viewform)
-- [Open a pull request](/docs/contribute/new-content/new-content/#fork-the-repo) with a new blog post. Create new blog posts in the [`content/en/blog/_posts`](https://github.com/kubernetes/website/tree/master/content/en/blog/_posts) directory.
-
-If you open a pull request, ensure that your blog post follows the correct naming conventions and frontmatter information:
+Anyone can write a blog post and submit it for review.
 
-- The markdown file name must follow the format `YYY-MM-DD-Your-Title-Here.md`. For example, `2020-02-07-Deploying-External-OpenStack-Cloud-Provider-With-Kubeadm.md`.
-- The front matter must include the following:
+### Guidelines and expectations
+
+- Blog posts should not be vendor pitches. 
+  - Articles must contain content that applies broadly to the Kubernetes community - For example, a submission should focus on upstream Kubernetes as opposed to vendor-specific configurations.
+  - Links should primarily be to the official Kubernetes documentation. When using external references, links should be diverse - For example a submission shouldn't contain only links back to a single company's blog.
```

Comment:
@castrojo - it might be helpful to put a link here to the Documentation Style Guide and "what's allowed" 

https://kubernetes.io/docs/contribute/style/content-guide/#what-s-allowed